<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExifGroupBox</class>
 <widget class="QGroupBox" name="ExifGroupBox">
  <property name="title">
   <string>Check Exif metadata</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <property name="checked">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Camera model</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Processing Software</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="HistoryComboBox" name="softwareComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="HistoryComboBox" name="cameraManufacturerComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Camera manufacturer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="HistoryComboBox" name="cameraModelComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Accepts regular expression</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/historycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
